tests/Adapter/OrderManagerTest.php

```diff
@@ -28,6 +28,9 @@ public function setUp(): void {
 	 * Test parsing Order for Student information
 	 */
 	public function testGetStudentFromOrder(): void {
+		global $dw_options;
+		$dw_options['useTkm'] = true;
```

Comment:
Should there be more tests for how the plugin should behave if `useTkm` is `false`?

Reply:
Yes, will add some